src/client/eppo-client.ts

```diff
@@ -462,11 +464,13 @@ export default class EppoClient {
     validateNotBlank(subjectKey, 'Invalid argument: subjectKey cannot be blank');
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
-    const { flag, configFetchedAt, configPublishedAt } = this.getFlagDetails(flagKey);
+    const flag = this.getFlag(flagKey);
+    const configDetails = this.getConfigDetails();
     const flagEvaluationDetailsBuilder = new FlagEvaluationDetailsBuilder(
+      configDetails.configEnvironment.name,
```

Comment:
Note to self: typescript ensures we don't need optional chaining here because of the environment interface in `ConfigDetails`, `configEnvironment: Environment;`